packages/backend/src/graphql/resolvers/peer.test.ts

```diff
@@ -212,7 +222,13 @@ describe('Peer Resolvers', (): void => {
             throw new Error('Data was empty')
           }
         })
-      await expect(gqlQuery).rejects.toThrow(ApolloError)
+      await expect(gqlQuery).rejects.toThrow(
+        new GraphQLError('unexpected', {
+          extensions: {
+            code: GraphQLErrorCode.InternalServerError
```

Comment:
I think if you change this to be a different enum value (e.g. `GraphQLErrorCode.Conflict`), the test will still pass. Maybe `.toThrow` isn't that smart :/. I think you'd have to check the extensions field in an explicit try-catch

Reply:
Yeah...it seems like the resolver won't throw a `GraphQLError` at the top level, but rather it gets caught and then added to a list of GraphQLErrors in a `ApolloError.graphQLErrors` field that ultimately gets thrown at the top level.